pkg/segment/results/mresults/metricresults.go

```diff
@@ -310,6 +310,15 @@ func (r *MetricsResult) Merge(localRes *MetricsResult) error {
 	return nil
 }
 
+func removeMetricNameFromGroupID(groupID string) string {
```

Comment:
add a comment above, what is the expected value of `groupID` and what is returned back